node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -622,6 +626,7 @@ impl Initialized {
 			},
 		}
 
+		drop(timer);
```

Comment:
Why is that drop necessary? Seems to be the end of the scope anyway?

Reply:
Explicit > Implicit. It helped me as as reasoned about the code.

Reply:
It is misleading though .. The implicit scoping thing is good because it will trigger no matter how you leave the scope. The explicit one will only trigger in the happy path ... so in general it is not telling you the whole story.